contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,299 @@
+                    },
+                },
+                {
+                    "dataset_name": "customer_table",
+                    "data": {
+                        "CUSTOMER_ID": [1, 2, 3],
+                    },
+                },
+            ],
+            "only_for": ["snowflake", "sqlite"],
+            "tests": [
+                {
+                    "title": "basic_positive_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "foreign_key_column": "CUSTOMER_ID",
+                        "foreign_table": "customer_table",
+                        "foreign_table_key_column": "CUSTOMER_ID",
+                    },
+                    "out": {
+                        "success": True,
+                        "result": {
+                            "observed_value": "0 missing values.",
+                            "unexpected_index_list": [],
+                        },
+                    },
+                },
+            ],
+        },
+        {
+            "data": [
+                {
+                    "dataset_name": "order_table_2",
+                    "data": {
+                        "ORDER_ID": ["aaa", "bbb", "ccc"],
+                        "CUSTOMER_ID": [1, 5, 6],
+                    },
+                },
+                {
+                    "dataset_name": "customer_table",
+                    "data": {
+                        "CUSTOMER_ID": [1, 2, 3],
+                    },
+                },
+            ],
+            "only_for": ["snowflake", "sqlite"],
+            "tests": [
+                {
+                    "title": "basic_negative_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "foreign_key_column": "CUSTOMER_ID",
+                        "foreign_table": "customer_table",
+                        "foreign_table_key_column": "CUSTOMER_ID",
+                    },
+                    "out": {
+                        "success": False,
+                        "result": {
+                            "observed_value": "2 missing values.",
+                            "unexpected_count": 2,
+                            "unexpected_index_list": [
+                                {"customer_id": "5"},
+                                {"customer_id": "6"},
+                            ],
+                        },
+                    },
+                },
+            ],
+        },
+    ]
+
+    @pydantic.root_validator
+    def _validate_template_dict(cls, values):
+        template_dict: dict = {
+            "foreign_key_column": values["foreign_key_column"],
+            "foreign_table": values["foreign_table"],
+            "foreign_table_key_column": values["foreign_table_key_column"],
+        }
+        values["template_dict"] = template_dict
+        return values
```

Comment:
Does this class have tests? How do we know these changes haven't broken things?

Reply:
Good call. I will add